### PR TITLE
[QAA-143] Improving Allure report

### PR DESCRIPTION
**Pull request by valber-ledger**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Improving Allure report:
- Adding Screenshot of the speculos display on failure

### ❓ Context

- **JIRA or GitHub link**: [QAA-143](https://ledgerhq.atlassian.net/browse/QAA-143)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[QAA-143]: https://ledgerhq.atlassian.net/browse/QAA-143?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/tests/utils/allureUtils.ts [hunk 1]
```diff
@@ -10,10 +12,19 @@ export async function addTmsLink(ids: string[]) {
...
 }
 
-export async function captureArtifacts(page: Page, testInfo: TestInfo) {
+export async function captureArtifacts(page: Page, testInfo: TestInfo, speculosApiPort: number) {
   const screenshot = await page.screenshot();
   await testInfo.attach("Screenshot", { body: screenshot, contentType: "image/png" });
 
+  try {
+    const response = await axios.get(`http://127.0.0.1:${speculosApiPort}/screenshot`, {
+      responseType: "arraybuffer",
```
Comment (abdurrahman-ledger) [inline apps/ledger-live-desktop/tests/utils/allureUtils.ts:21]: Maybe better to create and call a function in utils/speculos.ts ?

**Review by abdurrahman-ledger — COMMENTED** [review]
Result can be seen in this Allure [report](https://ledger-live.allure.green.ledgerlabs.net/allure/reports/8152c310-5959-4139-81e8-d8b18c192617/#suites/7969724dac228a6335e029ccd867d2fc/707fca95814c13a8/)

File: apps/ledger-live-desktop/tests/utils/allureUtils.ts [hunk 2]
```diff
@@ -2,6 +2,9 @@ import { allure } from "allure-playwright";
 import { Page, TestInfo } from "@playwright/test";
 import { promisify } from "util";
 import fs from "fs";
+import axios from "axios";
```
Comment (ypolishchuk-ledger) [inline apps/ledger-live-desktop/tests/utils/allureUtils.ts:5]: import is not in use

File: apps/ledger-live-desktop/tests/utils/allureUtils.ts [hunk 3]
```diff
@@ -14,6 +17,16 @@ export async function captureArtifacts(page: Page, testInfo: TestInfo) {
   const screenshot = await page.screenshot();
   await testInfo.attach("Screenshot", { body: screenshot, contentType: "image/png" });
 
+  try {
```
Comment (ypolishchuk-ledger) [inline apps/ledger-live-desktop/tests/utils/allureUtils.ts:20]: do we actually need try/catch here if it's already present in takeScreenshot(); method?

**Review by ypolishchuk-ledger — APPROVED** [review]

**Pull request merged**